pages/dashboard/Column.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * @file pages/dashboard/Column.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2003-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class Column
+ *
+ * @ingroup pages_dashboard
+ *
+ * @brief A class that represents a column in the submissions table
+ */
+
+namespace PKP\pages\dashboard;
```

Comment:
This should probably be somewhere in `classes`; the `pages` directory should only be used for handler classes.

Reply:
I've replaced it with anonymous class for now